### DISCUSSION TITLE
java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1968] [Q&A]
I continue with that giant case and today I went to add more items, but right from the start it gives the error above.
If I enter another output folder, the error does not occur.
So it could be that the many times I added things to the case, I may have damaged some file.

Any idea what this could be?
tks
Paulo

[IPED-2023-11-07-20-29-34.zip](https://github.com/sepinf-inc/IPED/files/13290302/IPED-2023-11-07-20-29-34.zip)

**Comment by lfcnassif** [comment]
A file related to the graph feature could have been corrupted. If you don't need the communication graph feature, delete the whole case/iped/graph (or neo4j I don't remember) folder and try to append the new evidence again.

**Reply by paulobreim** [reply]
Thank you @lfcnassif , it is working now. The folder is neo4j.
